src/components/SideDrawer/__test__/TestDrawerContents.test.tsx

```diff
@@ -1,14 +0,0 @@
-import { describe, expect, it } from 'vitest'
```

Comment:
Was this a delete or a rename? Looking now, the title seems redundant. 

`git mv` may be helpful in making the diff show the rename

Reply:
Ah, after fixing the vitest config, it is now okay not to import `describe`, `expect`, `it`, etc from `vitest` 🙇‍♀️